src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -120,7 +144,10 @@ public void Startup(StartupParams startupParams)
             LoadPackages(startupParams.Preferences, startupParams.PathManager);
         }
 
-        public void Ready(ReadyParams sp) { }
+        public void Ready(ReadyParams sp)
+        {
+            sp.CurrentWorkspaceChanged += OnCurrentWorkspaceChanged;
```

Comment:
@scottmitchell do you ever unsubscribe this handler?

Reply:
Nope. Just added the unsubscribe